### DISCUSSION TITLE
Installation ?

### DISCUSSION THREAD

**Discussion by RazorJah** [jhassan8/crunchyroll-tizen#26] [Q&A]
Hi,

Firstly: MANY THANKS !!!!!!!!!!!!!
I was waiting that since several month !!!

I'm not a developper, so it was a little hard for me to follow the tutoriel :D

In fine, i've been able to launch and test the application on my smart TV !
Works great !!!

But.... application was not installed, i've to relauch tizen studio and run the project again.

What i do badly ?

Regards,
Fabien

**Comment by jhassan8** [comment]
hi @RazorJah, I'm going to try making a video to make it easier... maybe this way won't work on all Tizen versions... I'll modify and add the other option... thanks for letting me know, regards.

**Comment by jhassan8** [comment]
the question disappeared, maybe he removed it, but there is now a guide with an installer in the [release](https://github.com/jhassan8/crunchyroll-tizen/releases)

**Comment by RazorJah** [comment]
Hello,

WAOOOOW
Many thanks, this guide and the .bat are really usefull for me !!!

It's now installed and works great !

Thank you for your work, i'm very impressed !

Best regards,
Fabien
